rpc/core/blocks.go

```diff
@@ -179,6 +179,17 @@ func BlockByHash(ctx *rpctypes.Context, hash []byte) (*ctypes.ResultBlock, error
 	return &ctypes.ResultBlock{BlockID: blockMeta.BlockID, Block: block}, nil
 }
 
+// TxStatus retrieves the status of a transaction given its hash. It returns a ResultTxStatus
+// containing the height and index of the transaction within the block.
+func TxStatus(hash []byte) (*ctypes.ResultTxStatus, error) {
```

Comment:
An explanation of the components of the hash would be beneficial. For example, is this related to the transaction key? What fields are included in the digest?